tools/build_pak_files.py

```diff
@@ -16,125 +16,125 @@
         unreal_editor_bin = os.path.realpath(os.path.join(args.unreal_engine_dir, "Engine", "Binaries", "Linux", "UE4Editor"))
         unreal_pak_bin    = os.path.realpath(os.path.join(args.unreal_engine_dir, "Engine", "Binaries", "Linux", "UnrealPak"))
 
-    unreal_project_dir         = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "cpp", "unreal_projects", "SpearSim"))
-    uproject                   = os.path.join(unreal_project_dir, "SpearSim.uproject")
-    unreal_project_content_dir = os.path.join(unreal_project_dir, "Content")
-
-    assert os.path.exists(args.contents_dir)
-    shared_dir = os.path.realpath(os.path.join(args.contents_dir, "Shared"))
-    scenes_dir = os.path.realpath(os.path.join(args.contents_dir, "Scenes"))
-
-    assert os.path.exists(shared_dir)
-    unreal_project_content_shared_dir = os.path.join(unreal_project_content_dir, "Shared")
+    # once we know target_platform, set our cooked dir
+    unreal_project_cooked_dir = os.path.realpath(os.path.join(unreal_project_dir, "Saved", "Cooked", target_platform + "NoEditor"))
 
     if spear.path_exists(unreal_project_content_shared_dir):
         print(f"[SPEAR | build_pak_files.py] Removing file or directory or symlink: {unreal_project_content_shared_dir}")
         spear.remove_path(unreal_project_content_shared_dir)
 
-    print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_shared_dir} -> {shared_dir}")
-    os.symlink(shared_dir, unreal_project_content_shared_dir)
+    print(f"[SPEAR | build_pak_files.py] Creating symlink: {unreal_project_content_shared_dir} -> {perforce_content_shared_dir}")
+    os.symlink(perforce_content_shared_dir, unreal_project_content_shared_dir)
 
-    assert os.path.exists(scenes_dir)
-    scene_names = [ os.path.basename(x) for x in os.listdir(scenes_dir) ]
+    assert os.path.exists(perforce_content_scenes_dir)
+    scene_names = [ os.path.basename(x) for x in os.listdir(perforce_content_scenes_dir) ]
     assert len(scene_names) > 0
 
     if args.scene_names is not None:
-        scene_content_dirs = [ os.path.realpath(os.path.join(scenes_dir, x)) for x in scene_names if fnmatch.fnmatch(x, args.scene_names) ]
-    else:
-        scene_content_dirs = [ os.path.realpath(os.path.join(scenes_dir, x)) for x in scene_names ]
-    assert len(scene_content_dirs) > 0
+        scene_names = [ s for s in scene_names if fnmatch.fnmatch(s, args.scene_names) ]
+
+    assert len(scene_names) > 0
+
+    for scene_name in scene_names:
+
+        pak_dirs = [
+            os.path.realpath(os.path.join(unreal_project_cooked_dir, "Engine", "Content")),
+            os.path.realpath(os.path.join(unreal_project_cooked_dir, "SpearSim", "Content", "Shared")),
+            os.path.realpath(os.path.join(unreal_project_cooked_dir, "SpearSim", "Content", "Scenes", scene_name)),
+        ]
+
+        txt_file = os.path.realpath(os.path.join(output_dir, scene_name + "_" + target_platform + ".txt"))
```

Comment:
Since pak file name has to end in `Win64` (based on build_executable.py script), we can use a separate variable for this.

Reply:
Fixed.